openshift-servicemesh/instance/base/jaeger2-elasticsearch.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger-prod2
```

Comment:
I have the same feedback for the control plane resource. I think it would be good to choose a completely generic name. Just `jaeger` might work or possibly something like `default` is better (since we already know it is a `Jaeger` resource). When we create variations as overlays, then perhaps we can get more specific with naming.